sr_world_generator/src/sr_world_generator/clean_world_file.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```

Comment:
output type hint missing

Reply:
Addressed in https://github.com/shadow-robot/sr_tools/pull/63/commits/21a28bc4b564f6a686881c813fe4a8e53bf3d7af